README.md

```diff
@@ -92,6 +92,62 @@ PIWIK_UPDATE_START_GOAL_ID | Matomo Update Start goal ID | 3
 SHOW_SERVICE_OFFLINE_PAGE | Feature Flag | false
 VF01_FORM_DOWNLOAD_URL | Overseas entity verification checks statement URL | `https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/1095139/OE_VF01.pdf`
 
+
+## Terraform and variables required
+
+Terraform is run via a concourse ci pipeline is used in place of AWS Cloud Formation to deploy the cluster and services.
```

Comment:
not sure if there is a word missing in this sentence, maybe an 'and', it didn't quite read right to me - Terraform is run via a concourse ci pipeline (and?) is used in place of AWS Cloud Formation

Reply:
Some bad editing when merging previous sentences,  added "and" then updated the sentence.